extensions/kafka/src/main/java/com/hazelcast/jet/kafka/KafkaSources.java

```diff
@@ -123,16 +139,18 @@ public static <K, V> StreamSource<Entry<K, V>> kafka(
      * @param projectionFn function to create output objects from the Kafka record.
      *                    If the projection returns a {@code null} for an item,
      *                    that item will be filtered out.
-     * @param topics the topics to consume, at least one is required
+     * @param topicsConfig configuration for the topics to consume, at least one
+     *                     topic must be specified in the configuration.
```

Comment:
The javadoc needs a more detailed update. Especially what's the behaviour when you set both offsets and a particular processing guarantee.